cli/errors.rs

```diff
@@ -156,12 +156,14 @@ fn get_url_parse_error_class(_error: &url::ParseError) -> &'static str {
 fn get_nix_error_class(error: &nix::Error) -> &'static str {
   use nix::errno::Errno::*;
   match error {
-    nix::Error::Sys(EPERM) => "PermissionDenied",
+    nix::Error::Sys(ECHILD) => "NotFound",
     nix::Error::Sys(EINVAL) => "TypeError",
     nix::Error::Sys(ENOENT) => "NotFound",
     nix::Error::Sys(ENOTTY) => "BadResource",
-    nix::Error::Sys(UnknownErrno) => unreachable!(),
-    nix::Error::Sys(_) => unreachable!(),
+    nix::Error::Sys(EPERM) => "PermissionDenied",
+    nix::Error::Sys(ESRCH) => "NotFound",
+    nix::Error::Sys(UnknownErrno) => "Error",
```

Comment:
👍  I get `Error: ESRCH: No such process` on Mac, but I think that's not what we'll get on Windows